packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -610,6 +633,15 @@ class RouteSettings {
 /// The type argument `T` is the corresponding [Route]'s return type, as
 /// used by [Route.currentResult], [Route.popped], and [Route.didPop].
 ///
+/// The [canPop] and [onPopInvoked] are used for intercepting pops.
+///
+/// {@tool dartpad}
+/// This sample demonstrates how to use this [canPop] and [onPopInvoked] to
```

Comment:
"this [canPop] and [onPopInvoked]" => "the [canPop] and [onPopInvoked] parameters"